api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/IdModelSelector.java

```diff
@@ -25,13 +27,29 @@ public IdModelSelector(String name) {
 
 	@Override
 	public Model getModel(HttpServletRequest request, ServletContext context) {
-		ModelMaker modelMaker = ModelAccess.on(context).getModelMaker(CONTENT);
+		ModelMaker modelMaker = ModelAccess.getInstance().getModelMaker(getRdfService());
 		VitroModelSource mSource = new VitroModelSource(modelMaker, context);
 		return mSource.getModel(name);
 	}
 
+	private WhichService getRdfService() {
+		if (isConfigurationModel()) {
+			return WhichService.CONFIGURATION;
+        	}
+		return CONTENT;
```

Comment:
What do you think about using ternary operator here?
```suggestion
		isConfigurationModel() ? return WhichService.CONFIGURATION : return CONTENT;
```

Reply:
Fixed